2023/AzureBengaluru/data.json

```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
+      "",
+      ""
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Bengaluru Azure Community",
+    "Country": "India",
+    "Url": "https://bengaluru.globalazure.in",
+    "SessionizeApiId": "10862",
+    "Description_SessionizeApiId": "If you have configured your Sessionize correctly then browsing to this url (replacing {Your sessionize event Id} with your event sessionize id) will return valid json: https://sessionize.com/api/v2/10862/view/Sessions",
+    "SessionizeEventName": "{Your sessionize event name (or complete URL to the public Sessionize C4P, i.e. https://sessionize.com/global-azure or just global-azure)}",
+    "Logo": {
+      "Description": "Community logo is a .png file 500X500px named AzureBengaluru.png.",
+      "File": "AzureBengaluru.png"
```

Comment:
This file is not present in your submission. Instead you have a file _template.png in the folder, and that is just a placeholder.